fe/fe-core/src/main/java/org/apache/doris/system/SystemInfoService.java

```diff
@@ -894,18 +893,8 @@ public long getBackendReportVersion(long backendId) {
     public void updateBackendReportVersion(long backendId, long newReportVersion, long dbId) {
         AtomicLong atomicLong = null;
         if ((atomicLong = idToReportVersionRef.get(backendId)) != null) {
-            Database db = Catalog.getCurrentCatalog().getDb(dbId);
-            if (db != null) {
-                db.readLock();
-                try {
-                    atomicLong.set(newReportVersion);
-                    LOG.debug("update backend {} report version: {}, db: {}", backendId, newReportVersion, dbId);
-                } finally {
-                    db.readUnlock();
-                }
-            } else {
-                LOG.warn("failed to update backend report version, db {} does not exist", dbId);
-            }
+            atomicLong.set(newReportVersion);
```

Comment:
In original implementation, we use db locks to ensure mutual exclusion and order.
So here we may still need to add table locks.

Reply:
atomicLong is AtomicLong,so I think it is meanless for add table lock or db lock?